schema/project-level/project-schema.json

```diff
@@ -554,6 +569,12 @@
       "title": "Delivery Location",
       "description": "The location where activity related to this project will be delivered, or will take place. A location may be described using a geometry (point location, line or polygon), a gazetteer entry, an address, or a combination of these.",
       "properties": {
+        "identifier": {
```

Comment:
Shouldn't this be `id`?

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/105/commits/53317e432480bc5fa8bee7e63cc1301fd0f7664d